lms/djangoapps/courseware/views.py

```diff
@@ -119,15 +119,19 @@ def courses(request):
     """
     Render "find courses" page.  The course selection work is done in courseware.courses.
     """
-    courses = get_courses(request.user, request.META.get('HTTP_HOST'))
-
-    if microsite.get_value("ENABLE_COURSE_SORTING_BY_START_DATE",
-                           settings.FEATURES["ENABLE_COURSE_SORTING_BY_START_DATE"]):
-        courses = sort_by_start_date(courses)
+    if settings.FEATURES.get('ENABLE_COURSE_DISCOVERY'):
+        # Send empty list for courses because they will be generated by course discovery app
+        return render_to_response("courseware/courses.html", {'courses', []})
```

Comment:
I get an error here. Maybe it is supposed to be {'courses': []} instead?

TypeError at /courses
unhashable type: 'list'
/edx/app/edxapp/edx-platform/lms/djangoapps/courseware/views.py in courses, line 124